src/DataSource/Types.hs

```diff
@@ -177,6 +179,7 @@ showURL :: ZendeskAPIUrl -> Text
 showURL AgentGroupURL                       = "users.json?role%5B%5D=admin&role%5B%5D=agent"
 showURL (UserRequestedTicketsURL userId)    = "/users/" <> toURL userId <> "/tickets/requested.json"
 showURL (UserAssignedTicketsURL userId)     = "/users/" <> toURL userId <> "/tickets/assigned.json"
+showURL (UserUnassignedTicketsURL)          = "/search.json?query=type%3Aticket%20assignee%3Anone&sort_by=created_at&sort_order=asc"
```

Comment:
We could use `urlEncode` function from  `http-types` to encode this.
https://hackage.haskell.org/package/http-types-0.12.1/docs/Network-HTTP-Types.html#v:urlEncode

So the query would look something like this:
`"/search.json?query=" <> decodeUtf8 (urlEncode False someQuery)`

I couldn't find url encoding function that returns `Text`, if there's one, that'll be better..

Reply:
Yeah, I wouldn't say it's practical to import a library for a single function. In this case, it's cleaner to write our own.